### PR TITLE
SAK-31345 Fix Hiding/unhiding "Site Info"

### PR DESCRIPTION
**Pull request by ansorgej**
SITE_VISIT is not in the list of permissions generated here for Site Info, for some reason.  Added a check to see if SITE_VISIT is in permissions list before trying to remove it.  If it's not there in the first place it probably doesn't need to be removed any more, right?

### CODE REVIEW

File: site-manage/pageorder/tool/src/java/org/sakaiproject/site/tool/helper/order/impl/SitePageEditHandler.java [hunk 1]
```diff
@@ -598,7 +598,7 @@ private boolean pageVisibilityHelper(String pageId, boolean visible, boolean ena
             if (!(permissions.isEmpty())) {
                 // We never change SITE_UPD at all.
                 permissions.remove(SITE_UPD);
-                permissions.remove(SITE_VISIT);
+                if (permissions.indexOf(SITE_VISIT) > 0) permissions.remove(SITE_VISIT);
```
Comment (bjones86) [inline site-manage/pageorder/tool/src/java/org/sakaiproject/site/tool/helper/order/impl/SitePageEditHandler.java]: Braces on single line conditionals please.

**Comment by ansorgej** [comment]
Whoops.  Sorry about that last commit.  Git-noob here.  I pushed a button I apparently shouldn't have.  Let me know if its a problem and I'll start fresh.

**Comment by jonespm** [comment]
It's not a big deal, github will squash the merges and rebase now. You should be able to remove it with a few git commands like.

git checkout <branch>
git reset <sha of original commit> -- This will put the changes back into an uncommitted state
git commit -a
git push <branch> origin --force

And then that redoes the entire commit. 


Generally to update the best process is to run
git pull --rebase 

or just run below and it will always do this when you git pull

git config branch.autosetuprebase always 

Then you don't get these extra commits in there.
http://stevenharman.net/git-pull-with-automatic-rebase

**Comment by ansorgej** [comment]
Thanks Matt!  I was clicking around in the Github desktop application without really knowing what I was doing or noticing which branch was active.  Before I knew it I'd done things that had been pushed up to my remote repository.  Duh.  Lesson learned.

Since my blunder didn't seem to actually introduce new changes to the PR and it'll get squashed anyway, I'm going to leave this branch/PR alone now unless someone suggests otherwise.

**Pull request merged**